homeassistant/components/denonavr/media_player.py

```diff
@@ -305,16 +310,24 @@ def media_episode(self):
         """Episode of current playing media, TV show only."""
         return None
 
+    @property
+    def is_dynamic_eq_enabled(self):
+        """Return boolean if DynamicEQ is currently enabled."""
+        return self._dynamic_eq
+
     @property
     def device_state_attributes(self):
         """Return device specific state attributes."""
+        state_attributes = {}
         if (
             self._sound_mode_raw is not None
             and self._sound_mode_support
             and self._power == "ON"
         ):
-            return {ATTR_SOUND_MODE_RAW: self._sound_mode_raw}
-        return {}
+            state_attributes[ATTR_SOUND_MODE_RAW] = self._sound_mode_raw
+        if self._dynamic_eq:
+            state_attributes[ATTR_DYNAMIC_EQ_ENABLED] = self._dynamic_eq
```

Comment:
The sound mode raw attribute is only added if the power is on, maybe we want the same for dynamic eq?

Probably best to just bail out with an empty dict if power is not `ON`.

Reply:
Good catch. Yes I will change that.